server/models/user.py

```diff
@@ -0,0 +1,15 @@
+from .shared import db, datetime
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(50), unique=True, nullable=False)
+    first_name = db.Column(db.String(50))
+    last_name = db.Column(db.String(50))
+    email = db.Column(db.String(80), unique=True)
+    timezone = db.Column(db.String(50), nullable=False)
```

Comment:
i think we should allow nullable because timezone won't be updated until the onboarding stages of the user creation